e2e/api/dataHelper.js

```diff
@@ -14,21 +15,37 @@ const getDate = days => {
   return date;
 };
 
+
 module.exports = {
   date: (days = 0) => {
     return getDateTimeISOString(days).slice(0, 10);
   },
 
-  dateNoWeekends: function dateNoWeekends(days = 0) {
-    let date = getDate(days);
-
-    if(date.getDay() !== 6 && date.getDay() !== 0) {
+  dateNoWeekends: async function dateNoWeekends(days = 0) {
+    const date = getDate(days);
+    let date_month = date.getMonth() + 1;
+    let date_date = date.getDate();
+    if (date_month.toString().length == 1) {
+      date_month = '0' + date_month;
+    }
+    if (date_date.toString().length == 1) {
+      date_date = '0' + date_date;
+    }
+    let date_String = date.getFullYear() + '-' + date_month + '-' + date_date;
+    let isDateABankHoliday = false;
+    try {
+      const rawBankHolidays = await fetch('https://www.gov.uk/bank-holidays.json');
+      const ukbankholidays = await rawBankHolidays.json();
+      isDateABankHoliday = JSON.stringify(ukbankholidays['england-and-wales'].events).includes(date_String);
+    } catch (err) {
+      console.log('Error while fetching UK Bank Holidays...', err);
```

Comment:
Minor: should be be `console.warn` since it could affect the test outcomes.

```suggestion
      console.warn('Error while fetching UK Bank Holidays...', err);
```

Reply:
done